docs/commands/status.md

```diff
@@ -38,6 +40,8 @@ netlify status:hooks
 **Flags**
 
 - `debug` (*boolean*) - Print debugging information
+- `httpProxy` (*option*) - Proxy server address to route requests through.
```

Comment:
For the other flags, the parenthesis is for the type, not the optionality, is it? I.e. should it say `(*string*)` instead.
Also, I would expect flags to be optional by default, and have `(required)` when not. What do you think?

Reply:
I found it confusing too, but it happens with other string options:
https://cli.netlify.com/commands/deploy/

I think `option` is the type.

I'll open a new issue to sort his.